packages/flutter/lib/src/widgets/page_view.dart

```diff
@@ -368,15 +369,26 @@ class _PagePosition extends ScrollPositionWithSingleContext implements PageMetri
   }
 
   double getPageFromPixels(double pixels, double viewportDimension) {
-    return math.max(0.0, pixels) / math.max(1.0, viewportDimension * viewportFraction);
+    final double actual = math.max(0.0, pixels) / math.max(1.0, viewportDimension * viewportFraction);
+    final int round = actual.round();
```

Comment:
there's a version of round that gives a double that might be better here

Reply:
fixed